pkg/keccak/contrib/Build/CompilerFlagsCortexM0.build

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<!--
+Implementation by the Keccak, Keyak and Ketje Teams, namely, Guido Bertoni,
+Joan Daemen, Michaël Peeters, Gilles Van Assche and Ronny Van Keer, hereby
+denoted as "the implementer".
+
+For more information, feedback or questions, please refer to our websites:
+http://keccak.noekeon.org/
+http://keyak.noekeon.org/
+http://ketje.noekeon.org/
+
+To the extent possible under law, the implementer has waived all copyright
+and related or neighboring rights to the source code in this file.
+http://creativecommons.org/publicdomain/zero/1.0/
+-->
+<group>
+
+    <fragment name="optimized">
```

Comment:
Seems to me that the content of these build files could be automated. The only thing that changes are the values of *arch*, *tune* (and *thumb* for native) and these values are actually in our CFLAGS already.

I just had a chat with @kaspar030 who suggest to generate this in the makefile with something like 

@echo "<group> <fragment name="optimized"><h>Common/brg_endian.h</h> <gcc>$(CFLAGS)</gcc> </fragment>"